src/components/Donate/Donate.js

```diff
@@ -19,41 +20,31 @@ class Donate extends Component {
 
     this.state = {
       search: '',
-      donations: [{
-        id: 1,
-        date: "23.02.2020",
-        amount: "140 mil.",
-        location: "РАЙОНЕН Ц-Р ПО ТРАНСФУЗИОННА ХЕМАТОЛОГИЯ - Стара Загора",
-        status: "Completed"
-      },
-      {
-        id: 2,
-        date: "12.08.2020",
-        amount: "310 mil.",
-        location: "РАЙОНЕН Ц-Р ПО ТРАНСФУЗИОННА ХЕМАТОЛОГИЯ - Стара Загора",
-        status: "Completed"
-      },
-      {
-        id: 3,
-        date: "23.12.2020",
-        amount: "",
-        location: "РАЙОНЕН Ц-Р ПО ТРАНСФУЗИОННА ХЕМАТОЛОГИЯ - Стара Загора",
-        status: "In Progress"
-      }
-      ]
+      loading: true,
+      donations: []
     }
 
     this.searchRef = React.createRef();
     this.donateDialogRef = React.createRef();
     this.donateButtonRef = React.createRef();
-
     this.search = this.search.bind(this);
     this.isSearched = this.isSearched.bind(this);
     this.filterDonations = this.filterDonations.bind(this);
     this.openDonateDialog = this.openDonateDialog.bind(this);
   }
 
-  componentDidMount() {
+  async componentDidMount() {
+    const url = BLOOD_BANK_API + '/donations/';
+    const response = await fetch(url, {
+      method: 'GET',
+    });
+    const data = await response.json();
+    console.debug(data);
```

Comment:
debug

Reply:
Changed it to console.log for consistency